zerver/tests/test_signup.py

```diff
@@ -3257,6 +3257,26 @@ def _assert_redirected_to(self, result: HttpResponse, url: str) -> None:
         self.assertEqual(result.status_code, 302)
         self.assertEqual(result["LOCATION"], url)
 
+    def signup(self, *,
```

Comment:
I'd call this something like `verify_signup`.

Reply:
Done